apps/test/unit/templates/RegionalPartnerSearchTest.js

```diff
@@ -76,4 +98,101 @@ describe('RegionalPartnerSearch', () => {
     server.respond();
     expect(wrapper.find('button')).to.have.length(0);
   });
+  it('shows no workshop cards if RP not found', () => {
+    createServerResponses(server, false, false);
+    const wrapper = shallow(<RegionalPartnerSearch {...MINIMUM_PROPS} />);
+    server.respond();
+    expect(wrapper.find('WorkshopCard')).to.have.length(0);
+  });
+  it('shows no workshop cards if RP is found but applications are closed', () => {
+    createServerResponses(server, true, true);
+    const wrapper = shallow(<RegionalPartnerSearch {...MINIMUM_PROPS} />);
+    server.respond();
+    expect(wrapper.find('WorkshopCard')).to.have.length(0);
+  });
+  it('shows no workshop cards if RP is found and applications are open, but RP is not offering programs', () => {
+    createServerResponses(server, true, false);
+    const wrapper = shallow(<RegionalPartnerSearch {...MINIMUM_PROPS} />);
+    server.respond();
+    expect(wrapper.find('WorkshopCard')).to.have.length(0);
+  });
+  it('shows workshop cards if RP found, applications are open, and RP is offering programs', () => {
+    createServerResponses(server, true, false, ['CSD', 'CSP']);
+    const wrapper = shallow(<RegionalPartnerSearch {...MINIMUM_PROPS} />);
+    server.respond();
+    expect(wrapper.find('WorkshopCard')).to.have.length(3);
+  });
+  it('shows Not Offering note on workshop card(s) for the program(s) not being offered when other programs are offered', () => {
+    createServerResponses(server, true, false, OFFERED_PROGRAMS, [
+      OFFERED_WORKSHOP
```

Comment:
This is great! And serves as documentation on how we expect data coming in from the backend to look